prow/jobs/kyma/components/ui-api-layer/ui-api-layer.jobs.yaml

```diff
@@ -1,30 +1,157 @@
+# general
+presets:
```

Comment:
Why do you put general configuration in job configuration file? This should be in config.yaml file

Reply:
removed - it was just for testing.